speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/RecorderTests.cpp

```diff
@@ -172,16 +172,17 @@ TEST(RecorderTests, RecorderThreeParamCompression)
 TEST(RecorderTests, WriteOnlyTime)
 {
     std::string folder = GetTestingDirectory();
+    DecisionTuple tuple;
     Recorder recorder(TEST_DIRECTORY, "test_recorder_time_only", 0, 0);
     recorder.WriteUserInput(nullptr, 0);
     recorder.WriteUserInput(nullptr, 2);
     recorder.WriteUserInput(nullptr, 1);
     recorder.WriteUserInput(nullptr, 6.9);
 
-    recorder.WriteDecisions(nullptr, 0);
-    recorder.WriteDecisions(nullptr, 3);
-    recorder.WriteDecisions(nullptr, 435);
-    recorder.WriteDecisions(nullptr, 95875);
+    recorder.WriteDecisions(tuple, 0);
+    recorder.WriteDecisions(tuple, 3);
+    recorder.WriteDecisions(tuple, 435);
+    recorder.WriteDecisions(tuple, 95875);
```

Comment:
This is no longer a "WriteOnlyTime" test now

Reply:
Implemented requested changed and made a new pr